civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -1723,6 +1731,32 @@ function civihr_employee_portal_menu() {
   return $items;
 }
 
+/**
+ * Callback function for 'emergency_contacts/{id}/delete
+ *
+ * @param string $id
+ *   The emergency contact ID
+ */
+function civihr_employee_portal_emergency_contact_delete($id) {
+  $service = new EmergencyContactDeletionNotifier();
+  $result = ['status' => 200];
+
+  try {
+    civicrm_api3('Contact', 'deleteemergencycontact', ['id' => $id]);
+    $service->notifyDeletion($id);
+  } catch (\Exception $exception) {
+    $result = [
+      'status' => 500,
+      'error' => $exception->getMessage(),
+    ];
+  }
+
+  drupal_add_http_header('Content-Type', 'application/json');
+  http_response_code($result['status']);
```

Comment:
Why was it necessary to change the response to be like this now?

Reply:
I just thought it would be nicer to have a properly functioning endpoint that returns a correct status code and JSON (since this is just used in AJAX requests) but if you think it's not necessary I can remove any output

Reply:
please remove it